src/Models/ClientOptions.php

```diff
@@ -31,7 +31,7 @@ class ClientOptions extends AuthOptions {
      * @var bool If true, msgpack is used for communication, otherwise JSON is used
      * note that msgpack is currently NOT SUPPORTED because of lack of working msgpack libraries for PHP
      */
-    public $useBinaryProtocol = false;
+    public $useBinaryProtocol = true;
```

Comment:
Can we start adding relevant spec references in the comments? Like [TO3f](https://docs.ably.io/client-lib-development-guide/features/#TO3f) can be mentioned in the doc comment.